alma/alma-installments-prestashop
# Feature v3.0.0 (In-Page, fix SEPA - purchase amount verification - return IPN refund mismatch)

### Reason for change



[Feature In-Page](https://linear.app/almapay/issue/MPP-213/deploy-in-page)
[Feature Flag In-Page](https://linear.app/almapay/issue/MPP-473/add-in-page-feature-flag-in-prestashop)
[Improve purchaseAmount verification](https://linear.app/almapay/issue/MPP-437/improve-purchaseamount-verification)
[Fix SEPA Payment](https://linear.app/almapay/issue/MPP-423/fix-issue-sepa)
[Change HTTP return code for mismatch for IPN](https://linear.app/almapay/issue/MPP-504/change-http-return-code-for-mismatch-for-ipn)

### Code changes


Everything with the refacto

### How to test

_As a reviewer, you are encouraged to test the PR locally._


- Activate In-Page
- Add feature flag in dashboard
- Make un refund mismatch and check the response HTTP manually if is 200

### Checklist for authors and reviewers




### Non applicable
